src/main/java/core/basesyntax/CalculatorOfArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface CalculatorOfArea {
```

Comment:
```suggestion
public interface AreaCalculator {
```

Reply:
@[zachar-chorny](https://github.com/zachar-chorny) zachar-chorny [yesterday](https://github.com/mate-academy/jv-oop-advanced/pull/631/commits/da67985868efed84a1f92026be27dae3437bcfea#r945248420)
Suggested change 
public interface IobtainArea {
public interface CalculatorOfArea {

Reply:
ok, not a big deal